pkg/certificates/certs.go

```diff
@@ -48,14 +49,14 @@ func createCertTemplate(expirationInterval time.Duration) (*x509.Certificate, er
 		NotBefore:             time.Now(),
 		NotAfter:              time.Now().Add(expirationInterval),
 		BasicConstraintsValid: true,
-		DNSNames:              []string{FakeDnsName},
+		DNSNames:              []string{san, FakeDnsName},
 	}
 	return &tmpl, nil
 }
 
 // Create cert template suitable for CA and hence signing
 func createCACertTemplate(expirationInterval time.Duration) (*x509.Certificate, error) {
-	rootCert, err := createCertTemplate(expirationInterval)
+	rootCert, err := createCertTemplate(expirationInterval, FakeDnsName)
```

Comment:
Will it create duplicated `FakeDnsName` in `DNSNames: []string{FakeDnsName, FakeDnsName}` in `createCertTemplate()`?

Reply:
Yes it will. This is at the CA Template where I am not sure why we have Subject Alternate Name values in the first place. I believe it can probably be removed in a later cleanup PR.